src/Traits/ReflectionFunctionLikeTrait.php

```diff
@@ -63,33 +70,59 @@ public function getClosureThis()
     {
         $this->initializeInternalReflection();
 
-        return forward_static_call('parent::getClosureThis');
+        return parent::getClosureThis();
     }
 
     public function getDocComment()
     {
+        if (!$this->functionLikeNode) {
+            $this->initializeInternalReflection();
+            return parent::getDocComment();
+        }
         $docComment = $this->functionLikeNode->getDocComment();
 
         return $docComment ? $docComment->getText() : false;
     }
 
     public function getEndLine()
     {
+        if (!$this->functionLikeNode) {
+            $this->initializeInternalReflection();
+            return parent::getEndLine();
+        }
         return $this->functionLikeNode->getAttribute('endLine');
     }
 
     public function getExtension()
     {
-        return null;
+        $extName = $this->getExtensionName();
+        if (!$extName) {
+            return null;
+        }
+        // The purpose of Go\ParserReflection\ReflectionExtension is
+        // to behave exactly like \ReflectionExtension, but return
+        // Go\ParserReflection\ReflectionFunction and
+        // Go\ParserReflection\ReflectionClass where apropriate.
+        return new ReflectionExtension($extName);
```

Comment:
Is it tested?

To actually test it you need to know what extension are loaded into PHP and try to reflect class provided by them.

Reply:
I included tests for two seemingly ubiquitous extensions. I believe I setup the tests to skip if not loaded... not sure I have coverage on this particular method, but I will verify. :+1: :memo: